shared/actions/fs.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+import * as Constants from '../constants/fs'
+import * as FsGen from './fs-gen'
+import * as I from 'immutable'
+import * as RPCTypes from '../constants/types/rpc-gen'
+import * as Saga from '../util/saga'
+import * as Types from '../constants/types/fs'
+import {posix as Path} from 'path'
```

Comment:
I don't think we should rename a thing as a known common module. people will assume Path is really path